src/docfx/build/metadata/ContributionInfo.cs

```diff
@@ -48,6 +47,8 @@ public ContributionInfo(Docset docset, GitHubUserCache gitHubUserCache)
             var authorNames = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
             var updatedDateTime = GetUpdatedAt(document, commits);
 
+            var resolveGitHubUsers = GithubUtility.TryParse(repo?.Remote, out var githubOwner, out var githubRepoName) && document.Docset.Config.GitHub.ResolveUsers;
```

Comment:
>resolveGitHubUsers [](start = 16, length = 18)

We will also support resolve github users for vsts repo. Add a TODO?

Reply:
I didn't change the logic here, just leave it to u later

---
In reply to: [224649997](https://github.com/dotnet/docfx/pull/3485#discussion_r224649997) [](ancestors = 224649997)